server/cmd/api/api.go

```diff
@@ -32,6 +32,9 @@ func (s *WebServer) Run() error {
 	auth.HandleFunc("/login", user.SignIn(s.db)).Methods("POST")
 
 	subrouter.HandleFunc("/users", user.List(s.db)).Methods("GET")
```

Comment:
Pode excluir essa rota que ela é inútil e transforma ela na de listar o usuário por id